src/TestCounter.php

```diff
@@ -30,12 +32,20 @@ public function countTotalTestsInFiles(array $files) : int
 
     public function countNumTestsInFile(string $file) : int
     {
-        $className = str_replace($this->testsDirectory, '', $file);
-        $className = str_replace('.php', '', $className);
-        $className = str_replace('/', '\\', $className);
-
         $numTestsInFile = 0;
 
+        $stream = new PHP_Token_Stream($file);
+        $classes = $stream->getClasses();
+
+        if (!count($classes))
+        {
+            return $numTestsInFile;
+        }
```

Comment:
Is `$classes` an array? If so, change this block of code to the following:

    if (!$classes) {
        return $numTestsInFile;
    }

If it is not an array, keep the count and move the opening { up to the same line as the if.